content/join/volunteer.md

```diff
@@ -40,3 +40,4 @@ Fill out the form below and we'll reach out with more details on you can help su
   {{< fields/disclaimer >}}
   {{< fields/submit label="Submit info" >}}
 </form>
+<script src="/assets/js/join.js"></script>
```

Comment:
Yeah it may be easier to include the JS in the location HTML file so that way you don't need to load any unneeded JS

Reply:
got it! I'll make those changes